services/Directory/data/FilterList.json

```diff
@@ -17371,5 +17371,16 @@
     "issuesUrl": "https://github.com/DandelionSprout/adfilt/issues",
     "licenseId": 35,
     "name": "Anti-'Stock vector art on copypaste backgrounds'"
+  },
+  {
+    "description": "Filter of malicious web resources (phishing, online fraud, malware, etc.) targeting Ukrainian citizens",
+    "donateUrl": "https://www.paypal.com/donate/?hosted_button_id=AQ9BB34WHFADC",
+    "homeUrl": "https://github.com/braveinnovators/ukrainian-security-filter",
+    "id": 2669,
+    "issuesUrl": "https://github.com/braveinnovators/ukrainian-security-filter/issues",
+    "licenseId": 4,
+    "name": "Ukrainian Security Filter",
+    "syntaxId": 3,
+    "viewUrl": "https://raw.githubusercontent.com/braveinnovators/ukrainian-security-filter/main/lists/adblock.txt"
```

Comment:
_viewUrl_ should be in [FilterListViewUrl.json](https://github.com/collinbarrett/FilterLists/blob/main/services/Directory/data/FilterListViewUrl.json)
The format is:
- filterListId: the ID here, 2669
- id: an incremental ID for each entry in ViewUrl
- primariness: If you specify more than one URL for your filterlist, this will indicate what order they should be used
- url: the URL of the filterlist

Thanks

Reply:
Already fixed